e3nn/networks.py

```diff
@@ -27,9 +31,16 @@ def __init__(self, Rs_in, Rs_hidden, Rs_out, lmax, layers=3, max_radius=1.0, num
 
         K = partial(Kernel, RadialModel=RadialModel, get_l_filters=partial(o3.selection_rule, lmax=lmax))
```

Comment:
```suggestion
        K = partial(Kernel, RadialModel=RadialModel, get_l_filters=partial(o3.selection_rule_in_out_sh, lmax=lmax))
```
Now you need to use this function for the selection of the SH that satisfies the parity